src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/Handlers/ProgressBar/ProgressBarTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Drawing;
+using System.Threading.Tasks;
+using Xamarin.Platform.Handlers.DeviceTests.Stubs;
+using Xunit;
+
+namespace Xamarin.Platform.Handlers.DeviceTests
+{
+	public partial class ProgressBarHandlerTests : HandlerTestBase<ProgressBarHandler>
+	{
+		[Fact]
+		public async Task ProgressInitializesCorrectly()
+		{
+			var progressBar = new ProgressBarStub()
+			{
+				Progress = 0.5
+			};
+
+			await ValidatePropertyInitValue(progressBar, () => progressBar.Progress, GetNativeProgress, progressBar.Progress);
+		}
+
+		[Fact]
+		public async Task ProgressColorInitializesCorrectly()
+		{
+			var progressBar = new ProgressBarStub()
+			{
+				ProgressColor = Color.Red
```

Comment:
Can this also be a theory to provide a few colors. At least the Red, Green, Blue and maybe the others like Yellow and Black and White. Single color tests I found might pass because of platform defaults. I know red is not common, but who knows what the future brings and iOS goes all red to celebrate Christmas Santas.

Reply:
Good point!. Changed the test to be a Theory with different values.